include/schrodinger/sketcher/sketcher_widget.h

```diff
@@ -480,6 +488,8 @@ class SKETCHER_API SketcherWidget : public QWidget
      */
     void onAtomHovered(const RDKit::Atom* atom);
     void onBondHovered(const RDKit::Bond* bond);
+
+    void onMolModelChanged(const bool molecule_changed);
```

Comment:
why is this needed at the Widget layer?

Reply:
I'm not quite sure what you mean.  It's not a signal or a public function; it's just a protected method that's used to respond to the MolModel's modelChanged signal.  We previously used a lambda for this, but the lambda was getting fairly long due to some logic I added, so I moved it to its own method.